src/redux/initialState.js

```diff
@@ -227,6 +227,44 @@ const initialState = {
   cart: {
     products: [],
   },
+  promo_products: [
+    {
+      id: 'aenean-ru-bristique-1',
+      name: 'Aenean Ru Bristique 1',
+      category: 'bed',
+      oldPrice: 123,
+      price: 30,
+      stars: 2,
+      newFurniture: true,
+      src:
+        'https://images.pexels.com/photos/1866149/pexels-photo-1866149.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
+      alt: 'sofa',
+    },
+    {
+      id: 'aenean-ru-bristique-2',
+      name: 'Aenean Ru Bristique 2',
+      category: 'bed',
+      oldPrice: 123,
+      price: 30,
+      stars: 2,
+      newFurniture: true,
+      src:
+        'https://images.pexels.com/photos/4505458/pexels-photo-4505458.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
+      alt: 'sofa',
+    },
+    {
```

Comment:
Why do you duplicate so much data :O Trust me, the only thing is needed here is an array with ids of promo products. Only ids. Like: [1, 6, 8]. Then it's up to you how you set `getAll` selector. Remember: selectors have access to the whole state, so you can easily prepare data with a map function to return detailed product info instead of ids. That way we'll have simple statePart with only ids and still get the detailed data we're looking for.